modules/nuxeo-apidoc-core/src/main/java/org/nuxeo/apidoc/snapshot/SnapshotManagerComponent.java

```diff
@@ -158,11 +156,17 @@ public DistributionSnapshot getSnapshot(String key, CoreSession session) {
         return snaps.get(0);
     }
 
+    protected Comparator<DistributionSnapshotDesc> getDistributionComparator() {
+        VersionComparator vc = new VersionComparator();
+        return Comparator.comparing(DistributionSnapshotDesc::getVersion, (d1, d2) -> {
+            return vc.compare(d2, d1); // reverse order
+        }).thenComparing(DistributionSnapshotDesc::getName);
```

Comment:
```suggestion
        return Comparator.comparing(DistributionSnapshotDesc::getVersion, new VersionComparator())
                         .reversed()
                         .thenComparing(DistributionSnapshotDesc::getName);
```

Reply:
If i remember well, lambda sorting does not properly handle null values, that's why i avoided it. It seems adding Comparator.nullsFirst is a workaround but i do not find it cleaner here

Reply:
Ok, but there's no lambda on my suggestion, I would say that if the proposed code has the issue, the committed one also has it. 
If we take a look at comparing or reversed methods, they use the Comparator#compare method, so if the NPE was about using a.equals(b), not sure we're exposed there.

Reply:
Ok i'll double-check (same for List#sort)

Reply:
Done, sorry, I got confused with unrelated lambda usage indeed, and I will make the change :+1: